src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+public class Main {
```

Comment:
One thing that isn't super obvious is that when we have a `module-info.java`, we can't have any classes in the "unnamed" package. So if you try to run/merge this with what is there depending on how your IDE behaves you will have some issues.